app/views/admin/base/dashboard.html.erb

```diff
@@ -3,14 +3,6 @@
   <h1>Admin Dashboard</h1>
 </div>
 
-<div class="admin-links">
-  <a class="btn admin_btn" href="<%= admin_categories_path %>">
-    <i class="fa fa-pie-chart fa-5x"><br>
-    </i>Category</a>
-  <a class="btn admin_btn" href="<%= admin_projects_path %>">
-    <i class="fa fa-book fa-5x"><br>
-    </i>Projects</a>
-  <a class="btn admin_btn" href="<%= admin_orders_path %>">
-    <i class="fa fa-files-o fa-5x"><br>
-    </i>Orders</a>
-</div>
+<h2>This is where the Platform Admin dashboard stuff will go.</h2>
```

Comment:
I don't think a ticket is needed because tests should be written as we go through user stories.